roles/lib_openshift/tasks/main.yml

```diff
@@ -1,5 +1,12 @@
 ---
 - name: lib_openshift ensure python-ruamel-yaml package is on target
   package:
-    name: python-ruamel-yaml
+    name: "{{ item }}"
     state: present
+  with_items:
+  - ruamel.yaml
+  - ruamel.yaml
+  - ruamel.yaml
+  - ruamel.yaml
+  - ruamel.yaml
+  - ruamel.yaml
```

Comment:
How this happened? Six times the same item? ``ruamel.yaml`` is not a package name. What was wrong with ``python-ruamel-yaml``?